### PR TITLE
blaster files updates

### PR DESCRIPTION
**Pull request by NoSloppy**

### CODE REVIEW

File: props/blaster_BC_buttons.h [hunk 1]
```diff
@@ -0,0 +1,797 @@
+/*
```
Comment (profezzorn) [inline props/blaster_BC_buttons.h:1]: seems like this should be a separate PR

**Comment by NoSloppy** [comment]
OK. I'll close and resubmit 2 separate PRs.
